src/llama_stack_provider_kft/kft_adapter.py

```diff
@@ -103,6 +142,72 @@ async def handler(
                 self.config.gpu_identifier: self.config.nproc_per_node,
             }
 
+            init_containers = None
+            logger.info(f"Preprocess: {self.config.preprocess}")
+            if self.config.preprocess:
+                dp_master_args = [
```

Comment:
Could that block be moved into a separate function to keep the handler function size reasonable?